### PR TITLE
Sidebar: Add toggle icons

### LINKED ISSUE #6005: New sidebar design implementation for the notes show page
Screen Shot 2019-07-11 at 12 05 38 PM

#### Checklist :

- [ ] Toggle icons - Add star and edit option from wikis and notes, And subpages icons @CleverFool77 
- [ ] Tags card and show two of them while hiding others @gautamig54 
- [ ] Add tags option
- [ ] Map and Add location button
- [ ] Add social media icons 

cc : @CleverFool77 @jywarren

### PR DESCRIPTION
**Pull request by CleverFool77**
Fixes #6005 part

Screenshot from 2019-07-12 16-37-59

Screenshot from 2019-07-12 16-38-19

### CODE REVIEW

**Comment by CleverFool77** [comment]
I removed the `Write a post button`, `Ask question` button and test related to it as it as not included in new design.
Thanks !!!

**Comment by CleverFool77** [comment]
'This system test for `location-modal` which is failing seems unrelated.

**Comment by CleverFool77** [comment]
I guess I need to do some refactoring on the way code is written because the screenshot is going well for note but not wiki.

**Comment by CleverFool77** [comment]
This is the diff in ss of both wiki and notes

#### WIKI
Screenshot from 2019-07-12 19-23-01

#### NOTE
Screenshot from 2019-07-12 19-23-15

But here I came across one problem in wiki. I'm getting this :thinking: and I checked it through console. And Right now trying the resolve and think the problem behind this.

Screenshot from 2019-07-12 19-26-11

cc : @gautamig54 @jywarren

**Comment by CleverFool77** [comment]
HI @gautamig54 @jywarren the code part of some portion of toggle icons is written in one cell whereas some other part is written in different cell which is leading to different amount of divs and thus distortion in UI. So Should i just refactor the whole code from all cells for toggle icons to one cell ? :thinking: or Should I just try to find a fix by letting it remain in different cells only.

**Comment by CleverFool77** [comment]
Hi @jywarren @gautamig54 @gauravano Any idea that what could be the reason for this ? 
The code part of unusual toggle icons on left of main title is not present in while checking through inspect element. Is it because of li mismatch ? :thinking: 
Screenshot from 2019-07-24 15-47-11

**Comment by CleverFool77** [comment]
Hi , So finally I found the reason of this extra toggle icons. And Now updated the PR.
These are the ss: 
### Wiki page
Screenshot from 2019-07-24 16-04-26

### Note page
Screenshot from 2019-07-24 16-05-39

I guess this PR is ready for review now.

**Comment by CleverFool77** [comment]
Hi @jywarren @gautamig54 I t would be great if you could review this.
Thanks !!

**Comment by jywarren** [comment]
This is great. Can you just in list form walk through the meaning of each icon, and are there tooltips for them all? Thank you!

**Comment by CleverFool77** [comment]
Hi @jywarren 
Yes!! They have tooltip and all the info that was being shown earlier is there in them.
ss attached : 
Screenshot from 2019-07-25 01-41-41

Screenshot from 2019-07-25 01-41-54
Screenshot from 2019-07-25 01-41-58

**Comment by jywarren** [comment]
Awesome. Can you make the circle edge a bit lighter? Like #ccc maybe? Then
this should be good to merge!!!

**Comment by CleverFool77** [comment]
Turned border color lighter to #ccc 

Screenshot from 2019-07-25 02-11-09

File: app/views/like/_like.html.erb [hunk 1]
```diff
@@ -1,92 +1,99 @@
...
+    
+    <br />
+    <hr style='clear:both;' />
+    
+    <div>
+      <p><a id='print-command-no-links'><i class='fa fa-print'></i> Print without page links</a></p>
+      <p><a id='print-command-3-col'><i class='fa fa-print'></i> Print in 3-column layout</a></p>
+    </div>
+    ">
+      <span class="ff fa fa-copy"></span>
```
Comment (jywarren) [inline app/views/like/_like.html.erb]: This is the only one I'm not sure of. Maybe this should be a `...` menu instead? I'm just worried the 2-pages icon isn't clear enough. What do you think?

Comment (CleverFool77) [reply app/views/like/_like.html.erb]: So What does that copy icon in style guide mean ? I thought copy icon looks like pages , so its something similar to print of 2 pages and other stuff menu.

Comment (CleverFool77) [reply app/views/like/_like.html.erb]: Or should I just remove copy icon from this and just have `. . .` menu ?

Comment (CleverFool77) [reply app/views/like/_like.html.erb]: Hi @jywarren The style guide doesn't seem to have this menu icon in sidebar ? Instead it's usually in main left column.

Screenshot from 2019-07-25 15-15-57

**Comment by jywarren** [comment]
Aha, you know what I think happened? I think the 3 sub-pages tooltip was
originally made on the 2-pages button, but moved for some reason. Sorry for
the confusion. I believe the 3 bullet points icon was originally for "table
of contents" actually! Let's just say that the dropdown menu for
"everything else" should just be (...), and we can fix the ambiguity in the
style guid as well. Apologies, and thanks!

**Comment by CleverFool77** [comment]
Ahhh So that's what it is. I'll fix this then and change it.
Thanks !!

**Comment by CleverFool77** [comment]
Lets have copy icon as subpages and all rest stuff in `bullet menu`. Is that right ?

**Comment by CleverFool77** [comment]
Hi @jywarren Just updated this PR.
screenshot attached: 

Screenshot from 2019-07-25 23-30-50
Screenshot from 2019-07-25 23-31-03
Screenshot from 2019-07-25 23-31-23

**Comment by jywarren** [comment]
looks awesome. Is this ready then? Thank you!

**Comment by CleverFool77** [comment]
Hi Jeff. Yes This PR is ready.
Thanks !!1

**Comment by jywarren** [comment]
Awesome!!

**Pull request merged**